babel.config.json

```diff
@@ -294,7 +301,8 @@
           "@department-of-veterans-affairs/platform-monitoring/DowntimeNotification/components/DowntimeApproaching": "./src/platform/monitoring/DowntimeNotification/components/DowntimeApproaching.jsx",
           "@department-of-veterans-affairs/platform-monitoring/DowntimeNotification/components/Wrapper": "./src/platform/monitoring/DowntimeNotification/components/Wrapper.jsx",
           "@department-of-veterans-affairs/platform-monitoring/DowntimeNotification/util/helpers": "./src/platform/monitoring/DowntimeNotification/util/helpers.js",
-          "@department-of-veterans-affairs/platform-monitoring/external-services/ExternalServicesError": "./src/platform/monitoring/external-services/ExternalServicesError.jsx"
+          "@department-of-veterans-affairs/platform-monitoring/external-services/ExternalServicesError": "./src/platform/monitoring/external-services/ExternalServicesError.jsx",
+          "@department-of-veterans-affairs/platform-monitoring/record-event": "./src/platform/monitoring/record-event.js"
```

Comment:
I think this can be cleaned up a lot as some functionality is exported from the `/exports` file.

So when you're importing it would be something like
```
import { recordEvent } from '@department-of-veterans-affairs/platform-monitoring/exports'
```

Reply:
noted, I was able to make minor changes, and removed a line or two from babel.config.json